### ISSUE TITLE
Default map position/zoom if location is unknown

### ISSUE THREAD

**Issue by ubruhin** [opacopac/navplan#20]
Hi, thanks for this awesome tool! :smiley: 

Somehow in Firefox the location provider does not seem to work for me (in Chromium it works on the same computer). Probably due to the unknown position, the map always centers and zooms on Bern when opening it. It's not a big deal, but still needs a few seconds every time to manually scroll/zoom to the location of interest.

Maybe it would be quite useful to store the last viewed map position and zoom level either in the user profile, or with cookies? Or as a simpler alternative, zoom on whole Switzerland instead of just Bern?

**Comment by opacopac** [comment]
hi mr bruhin!

thanks a lot! 👍 

actually navplan should work fine with firefox, i'm regularly using it myself. i assume some settings of firefox need to be updated in order to allow navplan to access the location provided by firefox. did you already try to edit the location settings in firefox?
![grafik](https://user-images.githubusercontent.com/13405736/104811647-4885c200-57fd-11eb-8e2b-994285cff0a8.png)

yes, your assumption is correct that LSZB is used as a fallback, when no location is provided by the browser. for the future it's actually planned to allow each user to configure its own home base. showing the whole CH in this case could be a good idea in the meanwhile. i will make some tests (especially with mobile screens) how it looks and might implement it that way. thanks for the input!

greets,
armand

**Comment by ubruhin** [comment]
> did you already try to edit the location settings in firefox?

Yes I checked that, the location access is allowed, no idea why it doesn't work...

> or the future it's actually planned to allow each user to configure its own home base. showing the whole CH in this case could be a good idea in the meanwhile. i will make some tests (especially with mobile screens) how it looks and might implement it that way.

Awesome, thanks! :+1: :smiley:
